pupil_src/shared_modules/pupil_producers.py

```diff
@@ -57,6 +57,24 @@ def plugin_menu_label(cls) -> str:
     def pupil_data_source_selection_label(cls) -> str:
         return cls.plugin_menu_label()
 
+    @staticmethod
+    def available_pupil_producer_plugins(g_pool) -> list:
+        def is_plugin_included(p, g_pool) -> bool:
```

Comment:
Since this is a closure, you can think about simplifying the call signature.
```suggestion
        def is_plugin_included(p) -> bool:
```

Reply:
true, but in general I try to not have references to outer scope inside function definitions. makes it easier to extract into a separate method later if needed.